src/object_finding/feature_descriptor.cpp

```diff
@@ -14,14 +14,15 @@ namespace lucidy{
         descriptor->setUpright( settings.calcUpright );
     }
 
-    DescriptorList& FeatureDescriptor::getDescriptorList(Image & sourceImage){
-        /// compute descriptors by using featureList that has been precalculated by detector
-        /// descriptor output will be placed in descriptorList
+    void FeatureDescriptor::computeDescriptors(Image & sourceImage){
+        /// compute descriptors by using image featureList that has been precalculated by detector
```

Comment:
You may add this in the extended Doxygen description (below @brief) within the header file.